concent_api/api_testing_helpers.py

```diff
@@ -80,5 +81,9 @@ def api_request(host, endpoint, private_key, public_key, data = None, headers =
     print()
 
 
+def get_strftime_from_timestamp(timestamp):
+    return datetime.datetime.fromtimestamp(timestamp).strftime("%Y-%m-%d %H:%M:%S")
```

Comment:
The name is misleading. Your function does not work like `strftime()` (which accepts an arbitrary format string, hence the `strf` in the name). It works more like [`isoformat()`](https://docs.python.org/3.6/library/datetime.html#datetime.datetime.isoformat), which uses a very specific format.

Please rename it to `timestamp_to_isoformat()`.

Actually, why not use `isoformat()` here?

Reply:
I'm using here strftime becuase I need time converted in `"%Y-%m-%d %H:%M:%S"` to freeze time to create message in specific time

Reply:
`strftime` does not freeze time. It just formats it.

And your format is exactly like the one `isoformat()` supports.